### PR TITLE
Convert slack auth page to widget, insert into org page when applicable

### PR DESCRIPTION
**Pull request by camhashemi**
@cvializ @andrewconner 

screen shot 2016-04-11 at 5 12 15 pm

screen shot 2016-04-11 at 5 03 53 pm

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/slack/SlackInfoCommander.scala [hunk 1]
```diff
@@ -81,7 +81,7 @@ trait SlackInfoCommander {
 
   // For generating OrganizationInfo
   def getOrganizationSlackInfo(orgId: Id[Organization], viewerId: Id[User], max: Option[Int] = None)(implicit session: RSession): OrganizationSlackInfo
-  def getOrganizationSlackTeam(orgId: Id[Organization], viewerId: Id[User])(implicit session: RSession): Option[OrganizationSlackTeamInfo]
+  def getOrganizationSlackTeam(orgId: Id[Organization])(implicit session: RSession): Option[OrganizationSlackTeamInfo]
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/SlackInfoCommander.scala:84]: @ryanpbrewster how important is the `viewerId` here? I see that it's not used to fetch anything, more just a guard against non-users viewing it. but the data is only `slackTeamId`, name, and last sync'd at.

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlack.tpl.html [hunk 2]
```diff
@@ -1,25 +1,4 @@
 <div class="kf-org-profile-slack">
   <div ui-view></div>
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlack.tpl.html:2]: @cvializ not sure how you're injecting the supplementary `keep` and `library` elements in here, but I'm curious :)

File: kifi-backend/modules/shoebox/app/com/keepit/slack/SlackInfoCommander.scala (hunk 1 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/SlackInfoCommander.scala:84]: Should be fine. I think it used to include libraries, so the `viewerId` was necessary to determine which libraries to show.

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlack.tpl.html (hunk 2 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlack.tpl.html:2]: should be from a child view of `orgProfile.slack` depending on what route you're in. see the [`orgProfile.slack` section of `route.js`](https://github.com/cvializ/keepit/blob/master/kifi-backend/modules/shoebox/angular/src/route.js#L125)

**Comment by cvializ** [comment]
coooooool 😮 👍 👍

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlack.tpl.html (hunk 2 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlack.tpl.html:2]: right, but I still don't get how those templates are being referenced and injected into `orgProfileSlack.tpl.html`

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlack.tpl.html:2]: you tell `ui-router` what child controllers/templates to use in `route.js` and it injects them into the parent route's `<div ui-view />` directive.

**Pull request merged**